cmfpy/algs/hals.py

```diff
@@ -1,80 +1,86 @@
 import numpy as np
 import numpy.linalg as la
 
-from .base import AbstractOptimizer, EPSILON
-from ..common import shift_and_stack, cmf_predict
+from .accelerated import AcceleratedOptimizer
+from ..common import shift_and_stack, EPSILON, FACTOR_MIN
 
 
-class SimpleHALSUpdate(AbstractOptimizer):
+class SimpleHALSUpdate(AcceleratedOptimizer):
```

Comment:
I feel like we should call this `AcceleratedHALS` or something... Or at least make `max_inner = 0` by default.

Reply:
I've set the default to a single iteration (i.e. regular HALS).